### ISSUE TITLE
Precise selection toggle in the Room Editor

### ISSUE THREAD

**Issue by Phoobs** [YoYoGames/GameMaker-Bugs#5156]
### Is your feature request related to a problem?

Currently in the room editor, objects that are layered on top of each other are difficult to select because invisible bounding boxes will often overlap.

### Describe the solution you'd like

I would like an option to toggle "precise" selection which would allow me to ignore bounding boxes when selecting things in the room editor and instead select objects by clicking the pixels of their sprite.

### Describe alternatives you've considered

The only reasonable alternate solution I've found is using seperate layers for objects that are grouped closely together, but this introduces significant layer clutter which ultimately just translates the same exact problem of workflow messiness to a different interface.

**Comment by nullbunny** [comment]
Yes please!! That would be so nice, not having to shuffle objects and sprites around constantly just to get to the one you want and then put everything back.
